hail/python/hailtop/pipeline/backend.py

```diff
@@ -336,17 +336,16 @@ def _cp(src, dst):
                 used_remote_tmpdir = True
             outputs += [x for r in task._external_outputs for x in copy_external_output(r)]
 
-            resource_defs = [r._declare(directory=local_tmpdir) for r in task._mentioned]
+            env_vars = {r._uid: r._get_path(local_tmpdir) for r in task._mentioned}
```

Comment:
Can you fix this for the local backend as well?

Reply:
I don't think it makes a big different in the local case because we'll either hit the number of arguments limit on the host (when we call `docker` from `sp.check_output`) or in the container (when we hit the max single argument length in bash). The value here, AFAICT, is that we're speaking to docker over HTTP, so we avoid the host or client shell entirely and go straight to the docker binary.